pio/src/iSpindel.cpp

```diff
@@ -1212,6 +1212,12 @@ void setup()
   initDS18B20();
   initAccel();
 
+  #ifdef WIFI_IS_OFF_AT_BOOT
```

Comment:
guess we can remove this guard, otherwise it will not be enabled now, right

Reply:
Not sure if it can be removed, as the function `WIFI_IS_OFF_AT_BOOT ` is not available in the API of older arduino versions.  `WIFI_IS_OFF_AT_BOOT` is set and introduced with v.3.0 when the persistence behaviour has changed and the function `WIFI_IS_OFF_AT_BOOT ` was added. The example in the docs (link above) show it like that.